test/containers/uploadButton.spec.jsx

```diff
@@ -1,84 +0,0 @@
-'use strict'
```

Comment:
It's not possible to reuse some of them to keep the uploadButton correctly tested? Or all of them are not useful anymore here?